testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -607,15 +606,17 @@ def reference(self):
         return u
 
     def test_iterative_average(self, mobile, reference):
-        res = align.iterative_average(mobile, niter=1)
-        res = align.iterative_average(mobile, reference, niter=1, eps=0)
+        res = align.iterative_average(mobile, niter=10)
         res = align.iterative_average(mobile, reference, eps=1e-5)
         res = align.iterative_average(mobile, reference, select='bynum 1:10',
                                       weights=np.ones(10))
 
+        with pytest.raises(RuntimeError):
+            _ = align.iterative_average(mobile, reference, niter=1, eps=0)
+
```

Comment:
Make this a separate test method `test_iterative_average_convergence_failure()`.

We want tests to be fine grained and testing one thing.